events/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from common.models import TimestampedModel, UUIDPrimaryKeyModel
+
+
+class Event(UUIDPrimaryKeyModel, TimestampedModel):
+    name = models.CharField(verbose_name=_("name"), max_length=64)
+    short_description = models.TextField(
+        verbose_name=_("short description"), blank=True,
+    )
+    description = models.TextField(verbose_name=_("description"), blank=True)
+    duration = models.IntegerField(verbose_name=_("duration"))
```

Comment:
`Event` should have a `ForeignKey` to `Venue`, right? :)

Reply:
According to the diagram, I believe event connects to an occurrence, which then connects to a venue, so basically an event can has multiple occurrences at different venues. Correct me if I'm wrong :D

Reply:
Yeah, you are right. Not sure which one would be actually better, but I guess we should go with the diagram, so please add venue foreign key to occurrence instead :)